app/boards/arm/bt60/bt60_v1.keymap

```diff
@@ -0,0 +1,180 @@
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+
+#define ANSI true
+//#define HHKB true
+//#define ISO true
+//#define ALL_1U true
+//#define SPLIT_BKSP_RSHFT true
+
+
+
+/ {
+	chosen {
+	#ifdef ANSI
+		zmk,matrix_transform = &ansi_transform;
+	#elif defined(HHKB)
+		zmk,matrix_transform = &hhkb_transform;
+	#elif defined(ISO)
+		zmk,matrix_transform = &iso_transform;
+	#elif defined(ALL_1U)
+		zmk,matrix_transform = &all_1u_transform;
+	#else
+		zmk,matrix_transform = &split_transform;
+	#endif
```

Comment:
Very neat! First time I've seen transform selection like this 👍

Reply:
Not my idea tbh, Pete suggested it to me